core/admin.py

```diff
@@ -138,7 +153,7 @@ def data_link(self, obj):
 
 
 @admin.register(models.ScoreAlgorithm)
-class ScoreAlgorithmAdmin(admin.ModelAdmin):
+class ScoreAlgorithmAdmin(AlgorithmAdmin):
     list_display = ('__str__', 'task', 'creator', 'created', 'active', 'data_link')
     readonly_fields = ('data_link',)
```

Comment:
Because we are now subclassing `AlgorithmAdim`, we can remove lines 143-151 (the entire class just contains the  `list_display` definition).